OAuth/Response/FacebookUserResponse.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\OAuth\Response;
+
+/**
+ * @author Guillaume Rodriguez <guiguidu31300@gmail.com>
+ */
+class FacebookUserResponse extends PathUserResponse
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getProfilePicture()
+    {
+        return 'https://graph.facebook.com/'.$this->getValueForPath('identifier').'/picture?width=600&height=600';
```

Comment:
There should be no hard coded URL. Doesn't `return $this->getValueForPath('profilepicture')` work?

Reply:
@XWB I just discovered a new feature in Graph API to directly specify the image size. Without this, the size of the avatar was 50x50, hence the hard URL.